Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -360,18 +367,20 @@ public class HTTPServer: Server {
 
      ### Usage Example: ###
      ````swift
-     let server = HTTPServer.listen(on: 8080, delegate: self)
+     let server = HTTPServer.listen(on: 8080, node: "localhost", delegate: self)
      ````
 
      - Parameter on: Port number for accepting new connections.
+     - Parameter address: The address of the network interface to listen on. Defaults to nil, which means this server
+                 will listen on all interfaces.
      - Parameter delegate: The delegate handler for HTTP connections.
 
      - Returns: A new instance of a `HTTPServer`.
     */
-    public static func listen(on port: Int, delegate: ServerDelegate?) throws -> ServerType {
+    public static func listen(on port: Int, address: String?, delegate: ServerDelegate?) throws -> ServerType {
```

Comment:
Same feedback here. This must match Kitura-net:
`public static func listen(on port: Int, address: String? = nil, delegate: ServerDelegate?) throws -> HTTPServer`